### DISCUSSION TITLE
blog/2022/uninstall-docker-on-macos

### DISCUSSION THREAD

**Discussion by giscus** [yqchilde/yqchilde.github.io#4] [Q&A]
# blog/2022/uninstall-docker-on-macos

当我从 Intel 芯片的 Mac 使用迁移助理迁移到 M1 芯片的 Mac 时，docker desktop 打不开了（尝试了很多次卸载重装M1版本的 docker desktop ），这是因为卸载 docker desktop 时没有卸载干净，即使是使用三方工具，如 sensei 工具卸载。这个文章就是为了解决这个问题。

https://yqqy.top/blog/2022/uninstall-docker-on-macos

**Comment by dayney** [comment]
写的不错哦，学习了
